### PR TITLE
[SQL] Generate SDK for Databases, ReplicationLinks  APIs

### PR DESCRIPTION
**Pull request by subhrob**
[SQL] Generate SDK for Databases, ReplicationLinks  APIs

- Generated SDK
- Updated AssemblyInfo and Management.Sql.csproj

### CODE REVIEW

**Comment by subhrob** [comment]
The Track2 changes are already published as part of https://github.com/Azure/azure-sdk-for-net/pull/30096
I would also like to validate using Track1

**Comment by subhrob** [comment]
Hi @ArthurMa1978 , could you please review this PR. It is a Track1 PR.
The Track2 changes are already published as part of https://github.com/Azure/azure-sdk-for-net/pull/30096 and we will use this in the future.
However, I would like to use this PR to be used in the existing powershell cmdlets and validate/ release the change.
My change involves addition of an extra enum type in an already existing enum.

**Review by ericshape — APPROVED** [review]
LGTM from SQL side

File: sdk/sqlmanagement/Microsoft.Azure.Management.Sql/src/Microsoft.Azure.Management.Sql.csproj [hunk 1]
```diff
@@ -7,14 +7,13 @@
 		<PackageId>Microsoft.Azure.Management.Sql</PackageId>
 		<Description>Azure SQL Management SDK library</Description>
 		<AssemblyName>Microsoft.Azure.Management.Sql</AssemblyName>
-		<Version>3.2.0-preview</Version>
+		<Version>3.2.1-preview</Version>
```
Comment (ArthurMa1978) [inline sdk/sqlmanagement/Microsoft.Azure.Management.Sql/src/Microsoft.Azure.Management.Sql.csproj:10]: Please also update the AzSdk.RP.props

Comment (subhrob) [reply sdk/sqlmanagement/Microsoft.Azure.Management.Sql/src/Microsoft.Azure.Management.Sql.csproj:10]: I have updated the AzSdk.RP.props

**Review by ArthurMa1978 — APPROVED** [review]

**Pull request merged**